### ISSUE TITLE
[Feature Request]Focus next entry after delete one

### ISSUE THREAD

**Issue by TroyDanielFZ** [Xyrio/RSSOwlnix#123]
As shown in the title.

OS: Win10
Version: 2.9.0-beta
Build Id: 2021-04-11

**Comment by Xyrio** [comment]
that works for me. give a more concrete example where you use it and how rss owl is configured (ungrouped? class?).

**Comment by TroyDanielFZ** [comment]
I'm sorry, I cannot repeat the problem again, after rebooting the computer.
Sorry for not enough testing before feedback.

**Comment by Xyrio** [comment]
maybe the preview didnt update and you thought it did not select the next one? like #50

**Comment by TroyDanielFZ** [comment]
According to my memory, which isn't always reliable, what I encountered is the same as [#50]. 

Let me recall my operations:
1. Filter the entries;
2. Click the first entry, finding it interesting;
3. Copy the descriptions/abstract for translation for skimming;
4. Done with current entry, delete it.
5. Found that the preview didn't update, and no entries in the list selected.
6. Press `Down` key, and one entry get selected ( but I forget which one get selected, just the next one, or the third one), and preview updated.

However, I cannot reproduce it. The phenomenon showed yesterday, but disappear after rebooting the computer.
I should have reboot the computer before report it as a bug.
Ah, 
> Reopening the program  solves 10% of the problems, and rebooting solves 99%.
